src/Contracts/MessageBuilderContract.php

```diff
@@ -0,0 +1,31 @@
+<?php
+namespace Softonic\TransactionalEventPublisher\Contracts;
+
+/**
+ * Interface MessageBuilderContract
+ *
+ * @package Softonic\TransactionalEventPublisher\Contracts
+ */
+interface MessageBuilderContract
+{
+    /**
+     * Builds the message to send to the event bus.
+     *
+     * @param       $eventType
+     * @param       $modelName
+     * @param       $payload
+     * @param array $metas
+     *
+     * @return mixed
+     */
+    public function build($eventType, $modelName, $payload, array $metas = []);
```

Comment:
It would be nice to have some type hinting if it applies (or specify mixed in the docs otherwise)

Reply:
I'm going to add type hinting when possible in the next commit. Thanks!